### ISSUE TITLE
driver.switch_to.active_element returns dictionary With Firefox but returns WebElement in Chrome

### ISSUE THREAD

**Issue by aaltat** [SeleniumHQ/selenium#4983]
## Meta -
OS:  
Ubuntu 17.10 

Selenium Version:  
Selenium 3.6.0 (Python 2)

Browser:  
Firefox and Chrome

Browser Version:  
Firefox:    56.0 (64-bit)
Chrome:  62.0.3202.62 (Official Build) (64-bit)

## Expected Behavior -
The `driver.switch_to.active_element` should work similar manner with Firefox and Chrome. I did expect the attribute to refer a `WebElement`

## Actual Behavior -
The Chrome returns `WebElement` but Firefox returns a dictionary which looks like this: 
```
{u'value': <selenium.webdriver.firefox.webelement.FirefoxWebElement (session="75f8062a-f4bc-4e80-ae2b-fc2a6e4327df", element="511996f9-b242-4c6c-bff9-7156c22a3823")>}
```

## Steps to reproduce -
```
from selenium import webdriver

gc = webdriver.Chrome()
gc.get('https://www.google.fi')
gc_active_element = gc.switch_to.active_element

ff = webdriver.Firefox()
ff.get('https://www.google.fi')
ff_active_element = ff.switch_to.active_element

print isinstance(gc_active_element, webdriver.remote.webelement.WebElement)
print isinstance(ff_active_element, dict)

gc.quit()
ff.close()
```

**Comment by lmtierney** [comment]
Hi, this was fixed in e649643 and should be in released version `3.7.0`

**Comment by aaltat** [comment]
Yes it is fixed 3.7.0, thanks from the support.
